styles/components/_list-group.scss

```diff
@@ -40,15 +40,15 @@
   > .menuitem > .icon { color: $brand-black; }
 
   > .menuitem > .icon:first-child {
-    margin-right: 10px;
+    width: 20px;
     // 16px is the default icon size
     // 4px = allowance
-    width: 20px;
+    margin-right: 10px;
   }
 
   > .menuitem > .badge { 
     float: right; 
-    background: $brand-primary;
+    background: $brand-emerald;
     color: $brand-white;
   }
```

Comment:
IMO, it's better to `@extend` `.badge.-emerald` instead of copy-pasting the variant rules